src/TraceEvent/Symbols/SymbolReader.cs

```diff
@@ -351,7 +351,7 @@ public ManagedSymbolModule OpenSymbolFile(string pdbFilePath)
                 if (firstBytes[0] == 'B' && firstBytes[1] == 'S' && firstBytes[2] == 'J' && firstBytes[3] == 'B')
                 {
                     stream.Seek(0, SeekOrigin.Begin);   // Start over
-                    ret = new PortableSymbolModule(this, pdbFilePath);
+                    ret = new PortableSymbolModule(this, stream, pdbFilePath);
```

Comment:
this is a minor bug fix: this method opens a stream, seeks to the begining of it but when it creates and instance of PortableSymbolModule it does not reuse the stream but instead calls a ctor that opens the file one more time:

https://github.com/microsoft/perfview/blob/ebc8046a8a70853fa1276089f2e0a07ec14f14fa/src/TraceEvent/Symbols/PortableSymbolModule.cs#L11
so far we always had the file opened twice